src/parser/ParsedQuery.cpp

```diff
@@ -40,7 +40,14 @@ string ParsedQuery::asString() const {
     // SELECT
     os << "\nSELECT: {\n\t";
     if(usesAsterisk) {
-      os << selectClause._varsOrAsterisk.getAsterisk();
+      auto list = selectClause._varsOrAsterisk.retrieveOrder();
+      std::list<string>::iterator it;
+      for (it = list.begin(); it != list.end(); ){
```

Comment:
`for (auto it = list.begin())`

(use `auto` for iterators, clang-tidy also recommends this.